NGINX Plus Release 34 (#343)
nginx/documentation: content/nginx/deployment-guides/single-sign-on/okta.md

---

> description: Enable OpenID Connect-based single sign-on (SSO) for applications proxied by NGINX Plus, using Okta as the identity provider (IdP).
> doctypes:
> - task

Reviewer: ```suggestion
description: Enable OpenID Connect-based single sign-on (SSO) for applications proxied by NGINX Plus, using Okta as the identity provider (IdP).
type:
- how-to
product: NGINX-PLUS
```
